Jenkinsfile

```diff
@@ -23,7 +23,7 @@ pipeline {
 	sh '''#!/bin/bash -ex
 	  source $HOME/bin/activate
 	  python3 setup.py develop
-	  TF_FORCE_GPU_ALLOW_GROWTH=true python3 -m pytest --cov=kymatio
+	  TF_FORCE_GPU_ALLOW_GROWTH=true python3 -m pytest --cov=kymatio --durations=0
```

Comment:
what is this line doing?

Reply:
Making Jenkins output the durations of the tests. It will be removed before merging (hence it being placed in a commit tagged “REMOVE”).

Reply:
OK!